Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -1193,6 +1193,26 @@ static AZStd::string ConcatPath(const char* szPart1, const char* szPart2)
     return ret;
 }
 
+#if defined(CARBONATED)
+static void cvar_OnAssertLevelCvarChanged(const int& assertLevel)
+{
+    CSystem::SetAssertLevel(assertLevel);
+}
+
+AZ_CVAR(
+    int,
+    sys_asserts,
+    1,
+    cvar_OnAssertLevelCvarChanged,
+    AZ::ConsoleFunctorFlags::IsCheat,
+    "0 = Suppress Asserts\n"
+    "1 = Log Asserts\n"
+    "2 = Show Assert Dialog\n"
+    "3 = Crashes the Application on Assert\n"
+    "Note: when set to '0 = Suppress Asserts', assert expressions are still evaluated. To turn asserts into a no-op, undefine "
+    "AZ_ENABLE_TRACING and recompile.");
+#endif
+
```

Comment:
Place this in a namespace (see rest of engine / project for examples). 

Suggest:

namespace Legacy::System::CVars
{
    AZ_CVAR(...)
}